spring-data-rest/src/main/test/com/baeldung/relationships/SpringDataRelationshipsTest.java

```diff
@@ -0,0 +1,103 @@
+package com.baeldung.relationships;
```

Comment:
This should be in src/test/java rather than src/main/test -- otherwise the Maven build won't run it